app/controllers/container_image_controller.rb

```diff
@@ -31,7 +31,7 @@ def openscap_rule_results
   def openscap_html
     @record = identify_record(params[:id])
 
-    send_data(@record.openscap_result.html, :filename => "openscap_result.html")
+    send_data(@record.openscap_result.html, :filename => @record.name.match(/((.*)\/)*(.*)$/i).captures[2].concat(".html"))
```

Comment:
regex nit: please use named capture so reader doesn't have to count parens.
also, consider a fallback in case the regex didn't match (for whatever reason, it's silly to crash in such case).

Reply:
hmm well, this regexp can't not match.  0 or more of something followed by anything :-).  so no need to add a fallback.